### PR TITLE
Fixed npm package name case

### PR DESCRIPTION
**Pull request by RobDesideri**
npm package name is in lowercase

### CODE REVIEW

File: htmlhint/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Integrates the [HTMLHint](https://github.com/yaniswang/HTMLHint) static analysis
 
 The HTMLHint extension will attempt to use the locally installed HTMLHint module (the project-specific module if present, or a globally installed HTMLHint module).  If a locally installed HTMLHint isn't available, the extension will use the embedded version (current version 0.9.13).
 
-To install a version to the local project folder, run `npm install --save-dev HTMLHint`.  To install a global version on the current machine, run `npm install --global HTMLHint`.
+To install a version to the local project folder, run `npm install --save-dev htmlint`.  To install a global version on the current machine, run `npm install --global htmlint`.
```
Comment (gregvanl) [inline htmlhint/README.md]: Good catch, but there is a typo in your fix (missing an 'h'), should be 'htmlhint'.

Comment (RobDesideri) [inline htmlhint/README.md]: Sorry!

**Comment by gregvanl** [comment]
@mike-kaufman small change to README when you publish again

**Comment by mike-kaufman** [comment]
@gregvanl  - do you know if I need to publish a new version to get the [marketplace page](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint) updated with this fix?  Or does it pull the readme directly from GitHub?

@RobDesideri  - thanks!!

**Comment by gregvanl** [comment]
@mike-kaufman You need to publish a new version of the extension. The README is pulled from the uploaded extension .vsix.

**Pull request merged**